XRFoundation/XREngine
# Material Library

Adds a bare bones material library and editor UI for overriding Model materials. 

To test: 
1. Pull and check out `material-component` branch
2. `npm run dev` and open any scene with a Model in the editor
3. Select the Model in the Hierarchy panel, and look for "Material Overrides" property. Add 1 override.
4. Select a Material from the dropdown. Leave the other options at default values
5. Click "Refresh" button. The model should now have a different shader on it.

Addresses #6224
